README.md

```diff
@@ -95,7 +95,7 @@ cmake .. -DCPPKAFKA_BUILD_SHARED=0
 
 # Using
 
-If you want to use _cppkafka_, you'll need to link your application with:
+If you want to use _cppkafka_, you'll need to include `cppkafka.h` header and link your application with:
```

Comment:
I'm not sure this clarification is needed. This is meant for users to know what they need to link to use the library. I would expect anyone who uses this C++ library knows they have to include some header to do so. Moreover, saying you "_need_ to include cppkafka.h" is not really accurate, you could include the individual headers instead.